ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,38 @@
 <!--
 Welcome to the go-ipfs bug tracker. This is for you! Please read, and then delete this text before posting it.
 
+If you're unfamiliar with how GitHub issues work, consider reading [GitHub's issues guide](https://guides.github.com/features/issues/).
+
 If you haven't yet searched the issue tracker for an existing report concerning your issue, please do so now.
 
 The go-ipfs issues are only for bug reports and directly actionable feature requests. Read https://github.com/ipfs/community/blob/master/contributing.md#reporting-issues if your issue doesn't fit either of those categories.
 
-If you have a *SUPPORT QUESTION*, please direct it to our forum at https://discuss.ipfs.io.
-
-Read https://github.com/ipfs/go-ipfs/blob/master/docs/github-issue-guide.md if you are not sure how to fill in this issue.
+* If you have a *SUPPORT QUESTION*, please direct it to our forum at https://discuss.ipfs.io.
+* If you want to discuss a design idea related to ipfs but not directly related to the go implementation, please file an issue at https://github.com/ipfs/ipfs.
 -->
 
 #### Version information:
 <!--
 Output From `ipfs version --all`
 
-If your version is older than *0.4.11* please update
-and re-check if the problem persists.
+If your version is older than *0.4.11* please update and re-check if the problem persists.
+
+If you can't run `ipfs version --all` or that command fails, include as much information as you can: IPFS version, computer architecture (e.g., Intel x86 64bit), operating system, etc.
 -->
 
 #### Type:
-<!-- Bug, Feature, Enhancement, Etc -->
+<!--
+
+- "bug": If what you are filing is a bug.
+- "feature": If you'd like to suggest a feature.
+- "enhancement ": If you'd like to suggest an improvement on to existing feature.
+- "test failure": If the tests are failing
+- "doc": If it's related to missing/incorrect documentation.
+- "meta": If its a meta issue about this project/this repository/issue tracker.
```

Comment:
You left out panic, I think that is useful for cases when ipfs crashes or other obvious bug.  A panic in my view indicates a programmer error.

Reply:
Ah. Good point. Delineates "I think this is a bug" and "this is definitely a bug".